spec/poros/food_total_hits_poro_spec.rb

```diff
@@ -0,0 +1,76 @@
+require 'rails_helper'
+
+RSpec.describe 'Food Total Hits Poro' do 
+  it 'it exists and has attributes' do 
+    data = {
```

Comment:
I appreciate that you shortened the stub a bit. If this were a project with more time, it'd be great to pull this into a fixture but for an assessment these PORO tests look great! For some of these trickier attributes, consider adding some tests/logic in your PORO around the absence  of some of the attributes